internal/scripts/execute.go

```diff
@@ -32,13 +32,13 @@ import (
 	"github.com/onflow/flow-cli/internal/command"
 )
 
-type flagsScripts struct {
```

Comment:
why this changed?

Reply:
These need to be exported to use in the other package